### PR TITLE
Mvcc Detach Delete

### PR DESCRIPTION
**Pull request by yiyun-sj**

### CODE REVIEW

File: src/include/storage/store/table.h [hunk 1]
```diff
@@ -141,6 +141,7 @@ class Table {
 
     common::TableType getTableType() const { return tableType; }
     common::table_id_t getTableID() const { return tableID; }
+    std::string getTableName() const { return tableName; }
```
Comment (ray6080) [inline src/include/storage/store/table.h:144]: Is this used anywhere?

File: src/storage/store/rel_table.cpp [hunk 2]
```diff
@@ -164,63 +155,72 @@ bool RelTable::delete_(Transaction* transaction, TableDeleteState& deleteState)
...
+    RelTableDeleteState* deleteState) {
+    KU_ASSERT(deleteState->srcNodeIDVector.state->getSelVector().getSelSize() == 1);
     const auto tableData =
         direction == RelDataDirection::FWD ? fwdRelTableData.get() : bwdRelTableData.get();
     const auto reverseTableData =
         direction == RelDataDirection::FWD ? bwdRelTableData.get() : fwdRelTableData.get();
     std::vector<column_id_t> columnsToScan = {NBR_ID_COLUMN_ID, REL_ID_COLUMN_ID};
-    const auto relIDVectors = std::vector<ValueVector*>{deleteState->dstNodeIDVector.get(),
-        deleteState->relIDVector.get()};
+    const auto relIDVectors = std::vector<ValueVector*>{&deleteState->dstNodeIDVector,
```
Comment (ray6080) [inline src/storage/store/rel_table.cpp:166]: `relIDVectors` doesn't seem to be an accurate name. I think you can directly initialize vectors into `relReadState->outputVectors`?

**Review by ray6080 — APPROVED** [review]
Thanks!

File: src/include/storage/store/table.h (hunk 1 above)
Comment (yiyun-sj) [reply src/include/storage/store/table.h:144]: yup, to get tableName for localTable during exception message

File: src/storage/store/rel_table.cpp (hunk 2 above)
Comment (yiyun-sj) [reply src/storage/store/rel_table.cpp:166]: sure

**Pull request merged**